### PR TITLE
Implement "status" command for Debian srptools init script

### PR DESCRIPTION
**Pull request by bvanassche**
As discussed on the linux-rdma mailing list, add support for querying the status of the srp_daemon(s) to the srptools init script.

### CODE REVIEW

File: debian/srptools.init [hunk 1]
```diff
@@ -9,83 +9,126 @@
...
-if [ "$PORTS" = "ALL" ]  ; then
-    for HCA_ID in `/bin/ls -1 ${IBDIR}`
-      do
-      for PORT in `/bin/ls -1 ${IBDIR}/${HCA_ID}/ports/`
-        do
-        run_daemon
-      done
-    done
-fi
+    STATUS=$(<"${IBDIR}/${HCA_ID}/ports/${PORT}/state")
```
Comment (jgunthorpe) [inline debian/srptools.init]: Failing to start the daemon due to port state is racey and will lead to boot up failures. Ultimately srp_daemon itself has to monitor the port state...

Comment (bvanassche) [inline debian/srptools.init]: The scope of this pull request is to add a status command to the Debian init script. The code your comment applies to is not new code but rework of existing code. In my opinion your comment falls outside the scope of this pull request. I will address port state monitoring separately.

Comment (jgunthorpe) [inline debian/srptools.init]: Seems reasonable

**Comment by jgunthorpe** [comment]
Our build process sets the execute bits during install so we could drop +x them from the git tree if desired.

**Comment by bvanassche** [comment]
Hello Doug, I have dropped the patch that sets executable mode for the Debian init script from this series.

**Pull request merged**